steps.md

```diff
@@ -0,0 +1,5 @@
+# create migrations
+docker-compose exec php-app bin/cake bake migration NameOfMigration
+
+# run migrations
+docker-compose exec php-app bin/cake migrations migrate
```

Comment:
I think you can utilize the README.md and add new section for your lesson learn.

Reply:
moved